frontend/package.json

```
@@ -1,81 +1,87 @@
 {
   "name": "customer-center",
-  "version": "1.1.0",
-  "description": "Small description for frontend goes here",
+  "description": "Rewrite of the never release Adfinis customer center.",
+  "repository": "https://github.com/adfinis-sygroup/customer-center",
+  "version": "2.0.0",
+  "license": "MIT",
   "private": true,
+  "author": {
+    "name": "Adfinis AG",
+    "email": "info@adfinis.com",
+    "url": "https://adfinis.com/"
+  },
+  "maintainers": [
+    "Akanksh Saxena <akanksh.saxena@adfinis.com>",
+    "Florian Mäder <florian.maeder@adfinis-sygroup.ch>"
```

i think this is already fixed?
```suggestion
    "Florian Mäder <florian.maeder@adfinis.com>"
```

---

We fixed it for the backend. I'll update it here as-well.

---

On the subject: Should I add you to these lists?

---

I corrected the address and added you to the list.